### PR TITLE
Release notes for v1.1.1.

### PR DESCRIPTION
**Pull request by wdschei**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
 # Releases #
 
+## Release 1.1.1 (2015-05-22) ##
+1. Added the new ApiCoverageHandler to log the path taken by a request to a logger named ```api-coverage-logger```.
+
 ## Release 1.1.0 (2015-04-15) ##
 1. Classes have been repackaged into different packages.
 This breaks backwards compatibility for classes related to Result and Step
-2. Added support for rax:captureHeader extension.  If added to a wadl param, it will put the value
```
Comment (RackerWilliams) [inline RELEASE.md]: You don't need to modify this stuff...

**Comment by RackerWilliams** [comment]
Also take a look at https://github.com/rackerlabs/api-checker/issues?q=milestone%3A1.1.1+is%3Aclosed

We also fixed a bug where tools were not in their correct package.

**Comment by wdschei** [comment]
Added the CLI tools fix and removed the bullet mods.

File: RELEASE.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Releases #
 
+## Release 1.1.1 (2015-05-22) ##
+1. Added the new ApiCoverageHandler to log the path taken by a request to a logger named ```api-coverage-logger```.
+1. Moved the Wadl2Checker and Wadl2Dot command line utilities out of the test source tree into the main source tree.
```
Comment (RackerWilliams) [inline RELEASE.md]: How about saying, fixed a bug where CLI utilities were accidentally placed in test source tree :-)

Comment (wdschei) [inline RELEASE.md]: Okey Dokey.

**Comment by RackerWilliams** [comment]
:+1: Looks good.  You can release when ready.

**Pull request merged**